### PR TITLE
Added structure for topic creation with docs

### PR DESCRIPTION
**Pull request by HumairAK**
Related: [this](https://github.com/operate-first/odh-moc-support/issues/27) and #10

### CODE REVIEW

File: docs/add_kafka_topics.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  name: orders
+  labels:
+    strimzi.io/cluster: odh-message-bus
+spec:
+  partitions: 2
+  replicas: 3
+```
+
+The label `strimzi.io/cluster` should have the value `odh-message-bus`. The recommended partition size is `2` and
+replica count is `3`. Pick a suitable name, ensure that it's unique in the `topics` folder.
```
Comment (maulikjs) [inline docs/add_kafka_topics.md:22]: You might also want to mention here that the replica count should be lower than/equal to the number of Kafka brokers

Comment (HumairAK) [reply docs/add_kafka_topics.md]: updated, thanks!

**Comment by maulikjs** [comment]
Looks good now

**Review by anishasthana — APPROVED** [review]
:ship:

**Pull request merged**